hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/TransferOmLeaderSubCommand.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.hadoop.ozone.admin.om;
+
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.ozone.om.protocol.OzoneManagerProtocol;
+import picocli.CommandLine;
+import java.util.concurrent.Callable;
+
+
+/**
+ * Handler of ozone admin om transfer command.
+ */
+@CommandLine.Command(
+    name = "transfer",
+    description = "Manually transfer the raft leadership to the target node.",
+    mixinStandardHelpOptions = true,
+    versionProvider = HddsVersionProvider.class
+)
+public class TransferOmLeaderSubCommand implements Callable<Void> {
+
+  @CommandLine.ParentCommand
+  private OMAdmin parent;
+
+  @CommandLine.Option(
+      names = {"-id", "--service-id"},
+      description = "Ozone Manager Service ID."
+  )
+  private String omServiceId;
+
+  @CommandLine.ArgGroup(multiplicity = "1")
+  private TransferOption configGroup;
+
+  static class TransferOption {
+    @CommandLine.Option(
+        names = {"-n", "--nodeId"},
```

Comment:
Use `transfereeId` or `newLeaderId`.

BTW, is there a way to specify the which OM to send the command?

Reply:
> BTW, is there a way to specify the which OM to send the command?

This is the client's function. Currently, it seems not supported.